workers/data_refinery_workers/processors/utils.py

```diff
@@ -401,11 +396,13 @@ def run_pipeline(start_value: Dict, pipeline: List[Callable]):
 
 class ProcessorJobError(Exception):
     """ General processor job error class. """
-    def __init__(self, failure_reason, *, success=None, no_retry=None, **context):
+    def __init__(self, failure_reason, *, success=None, no_retry=None, retried=None, abort=None, **context):
```

Comment:
Wouldn't it make more sense for `abort` to default to False than None?

Reply:
I used `None` to differentiate when the user wants to explicitly set a value for the job in the database. For example: `ProcessorJobError('message', abort=False)` will set `job.abort=False`, on the other hand `ProcessorJobError('message')` would leave the `job.abort` to the default value.